host/sgx/linux/xstate.c

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "../xstate.h"
+#include <openenclave/internal/sgxtypes.h>
+#include <openenclave/internal/trace.h>
+#include "../cpuid.h"
+
+/*
+**==============================================================================
+**
+** Implementation of SGX Platform Capability Discovery.
```

Comment:
Again, update comment for clarity. The xstate feature discovery here is not specific to SGX platforms and this file does not actually detect SGX features from CPUID (as does happen in oesgx). Consider just removing this block comment entirely, the rest of the embedded comments do a better job describing what is done.

Reply:
ok